src/content/docs/new-relic-control/agent-control/troubleshooting.mdx

```diff
@@ -0,0 +1,116 @@
+
+Forward a local port to the main agent-control pod:
+```shell
+kubectl port-forward <pod-name> 51200:51200
+```
+Request the agent status:
+```shell
+curl localhost:51200/status
+```
+## Helm release failure
+Agent Control requires a valid authentication credential to securely connect to Fleet Control. Initially, this credential is automatically generated through the Agent Control installation UI and is represented by the `identityClientId` and `identityClientSecret` fields in the values file.
+
+For security reasons, the credential necessary for installing Agent Control expires after 12 hours. If the installation fails with the error message below, follow these troubleshooting steps:
+```shell
+Error: UPGRADE FAILED: pre-upgrade hooks failed: job failed: BackoffLimitExceeded
+```
+
+Check the logs of the Kubernetes job in charge of setting up the credential:
+
+First, identify the job’s pods:
+```shell
+kubectl describe job agent-control-system-identity -n <your-namespace>
+```
+
+The Events section will point to the specific pods, as follows:
+```shell
+Events:
+      Type     Reason                Age   From            Message
+      ----     ------                ----  ----            -------
+      Normal   SuccessfulCreate      88s   job-controller  Created pod: agent-control-system-identity-installer-jr6cg
+      Normal   SuccessfulCreate      73s   job-controller  Created pod: agent-control-system-identity-installer-wnx2v
+      Normal   SuccessfulCreate      50s   job-controller  Created pod: agent-control-system-identity-installer-8zsqd
+      Normal   SuccessfulCreate      7s    job-controller  Created pod: agent-control-system-identity-installer-btqh7
+      Warning  BackoffLimitExceeded  1s    job-controller  Job has reached the specified backoff limit
+```
+
+You can view the logs of the corresponding pods as follows:
+```shell
+kubectl logs <pod-name> -n <your-namespace>
+```
+
+Here's an example:
+```shell
+kubectl logs agent-control-system-identity-installer-btqh7 -n newrelic
+```
+
+Retry the installation via Helm and check the logs. Here're some known issues:
+
+- **Invalid `identityClientId`:** `Error getting system identity auth token. The API endpoint returned 404: Failed to find Identity: <identityClientId-value>`
+- **Invalid `identityClientSecret`:** `Error getting system identity auth token. The API endpoint returned 400: Bad client secret.`
+- **Identity expired:** `Error getting system identity auth token. The API endpoint returned 400: Expired client secret.`
+- **Missing required permissions:** `Failed to create a New Relic System Identity for Fleet Control communication authentication. Please verify that your User Key is valid and that your Account Organization has the necessary permissions to create a System Identity: Exception while fetching data (/create) : Not authorized to perform this action or the entity is not found.`
```

Comment:
The page looks weird

<img width="639" alt="Screenshot 2025-02-19 at 18 16 14" src="https://github.com/user-attachments/assets/e6bb5765-0039-4ec7-aa4f-9ec95dfaa94c" />